src/Service.ts

```diff
@@ -154,16 +183,19 @@ export default class Service {
 
   public async update() {
     await _post({
-      errorWithCleanerStack: getErr(`Failed to update service ${this._name} of dataset ${this._datasetNameWithOwner}.`),
+      errorWithCleanerStack: getErr(
+        `Failed to update service ${this._name} of dataset ${(await this._dataset.getInfo(true)).name}.`
+      ),
       app: this._app,
       path: await this._getServicePath(),
-      data: { recreate: true },
+      // works with SV1????
```

Comment:
I think this is a superfluous comment

Reply:
that being said, did you run the testsuite on a v1 instance as well?

Reply:
I have now. All good. I've actually modified a few unit tests to account for SV1 or SV2.